deploy/gk-deploy

```diff
@@ -294,7 +294,7 @@ assign() {
 #   The following resource types have unique conditional states:
 #
 #   * Pods
-#     - default: state reports '1/1'
+#     - default: state reports 'ready/desired' where `ready == desired`, eg. '1/1'.
```

Comment:
Change this to: `the desired number of containers are ready.`

Reply:
done! thanks.